Scripts/Spec/GLTFNode.cs

```diff
@@ -62,7 +62,7 @@ public class ImportTask : Importer.ImportTask<ImportResult[]> {
 				this.meshTask = meshTask;
 				this.skinTask = skinTask;
 				this.cameras = cameras;
-				task = new Task(() => { });
+				//task = new Task(() => { });
```

Comment:
This change has broken the Unity editor import process. At least for a project being opened on a new machine. Importer.cs line 161 calls task.RunSynchronously and task is null.